app/views/layouts/promo_registrations.html.slim

```diff
@@ -3,15 +3,16 @@
 
 - content_for(:nav) do
   .nav.nav--promo
-    = link_to(root_path) do
-      .brave-logo
-        = render "logo_wordmark_white_svg"
+    / = link_to(root_path) do
+    /   .brave-logo
```

Comment:
Can we just remove these if we're not going to use them?

Reply:
https://github.com/brave-intl/publishers/pull/2333/commits/86c918bf002a417ed15a74c395eb8246a7429b96